vendor/github.com/RangelReale/osincli/README.md

```
@@ -0,0 +1,77 @@
+
+Golang OAuth2 client library
+----------------------------
+
+OSINCLI is an OAuth2 client library for the Go language, as specified at
+http://tools.ietf.org/html/rfc6749.
+
+Using it, you can access an OAuth2 authenticated service.
+
+The library follows the RFC recommendations, but allows some differences, like passing the client secret in the url instead of using the Authorization header.
+
+### Example
```

Markdownlint (0.37.0)

Expected: setext; Actual: atx
Heading style

(MD003, heading-style)